mmcv/parallel/collate.py

```diff
@@ -72,7 +72,7 @@ def collate(batch, samples_per_gpu=1):
                 stacked.append(
                     [sample.data for sample in batch[i:i + samples_per_gpu]])
         return DataContainer(stacked, batch[0].stack, batch[0].padding_value)
-    elif isinstance(batch[0], Sequence):
+    elif isinstance(batch[0], Sequence) and not isinstance(batch[0], str):
```

Comment:
Add a comment to describe why we need to check whether `batch[0]` is a string. In addition, the issue link can be provided here like `More details at https://github.com/open-mmlab/mmcv/issues/1387.`